lib/faraday/request.rb

```diff
@@ -24,13 +24,14 @@ module Faraday
   #   @return [String] body
   # @!attribute options
   #   @return [RequestOptions] options
-  #
-  # rubocop:disable Style/StructInheritance
-  class Request < Struct.new(:http_method, :path, :params, :headers, :body, :options)
-    # rubocop:enable Style/StructInheritance
-
+  Request = Struct.new(:http_method, :path, :params, :headers, :body, :options) do
     extend MiddlewareRegistry
 
+    alias_method :member_get, :[]
+    private :member_get
```

Comment:
`member_get` doesn't seem to be used in this file. Is it still necessary?

Reply:
This is a leftover from an earlier iteration, it could be removed.

Reply:
Good point. I'll merge this as-is and cleanup in my other PR 👍